### PR TITLE
Home and product page

### PR DESCRIPTION
**Pull request by tanveertkd**
This PR contains the home and product page for the e-com website.
The images might not load up because of the resolution, clicking will open up the images or please click on the link below to see the live project.

Home  
  
Product Page  
  
Product Page - Mobile

### CODE REVIEW

File: css/colors.css [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-    /* --color-light-shadow: #f8cfa9; */
     --color-hover-grey: #424242;
-    --color-hover-light-grey: #dbd7d7;
+    --color-light-grey: #bdbdbd;
+    --color-lighter-grey: #dbd7d7;
     --color-highlight-orange: #ff3d00;
-    --color-text-lighter-grey: #bdbdbd;
+
+    --color-grey-light: #E6E6EA;
+    --color-grey-lighter: #F4F4F8;
```
Comment (BelideNikhil) [inline css/colors.css:11]: Names these as light-grey-100, light-grey-300, light-grey-500, 100 meaning very light and so on.

**Review by BelideNikhil — APPROVED** [review]
Also maybe try writing all the media queries at one places as youare switching only at 2 levels, 786 and 1024.
Rest all looks good!
Great job!

**Review by kiranpratyush — APPROVED** [review]
Please consider changing variable names as pointed out

File: css/reset.css [hunk 2]
```diff
@@ -1,3 +1,8 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
 html, body {
     margin: 0;
     padding: 0;
```
Comment (sakshamAK) [inline css/reset.css:8]: you can remove this as you're using * I guess

**Review by sakshamAK — APPROVED** [review]

**Review by Swapnilkota108 — APPROVED** [review]

File: css/reset.css (hunk 2 above)
Comment (tanveertkd) [reply css/reset.css:8]: Thanks for pointing that out!

File: css/colors.css (hunk 1 above)
Comment (tanveertkd) [reply css/colors.css:11]: I'll definitely look into this, thanks!

**Comment by tanveertkd** [comment]
> Also maybe try writing all the media queries at one places as youare switching only at 2 levels, 786 and 1024. Rest all looks good! Great job!

I'm actually trying to keep all the styling related to each components separate.

**Pull request merged**